### ISSUE TITLE
Instruments prefixed with '+' result in parser error

### ISSUE THREAD

**Issue by mesca** [csound/csound#1584]
In Csound 7, an instrument definition such as `instr +MyInstrument` produces the following error:

```
error: syntax error, unexpected '+', expecting NEWLINE or T_IDENT or T_PLUS_IDENT or INTEGER_TOKEN  (token "+")
```

**Comment by vlazzarini** [comment]
Looks like that rule is not yet implemented. Any suggestions as to how we should implement this @kunstmusik?

**Comment by vlazzarini** [comment]
fixed in commit a44f7a16794767
added test to suite.

**Comment by mesca** [comment]
Thanks! I confirm it now works as expected.
